dist/svg.js

```diff
@@ -2414,15 +2415,28 @@ SVG.Matrix = SVG.invent({
     }
     // Skew
   , skew: function(x, y, cx, cy) {
-      return this.around(cx, cy, this.native().skewX(x || 0).skewY(y || 0))
+      // support uniformal skew
+      if (arguments.length == 1) {
+        y = x
+      } else if (arguments.length == 3) {
+        cy = cx
+        cx = y
+        y = x
+      }
+
+      // convert degrees to radians
+      x = SVG.utils.radians(x)
+      y = SVG.utils.radians(y)
+
+      return this.around(cx, cy, new SVG.Matrix(1, Math.tan(y), Math.tan(x), 1, 0, 0))
```

Comment:
When you chain skewX with skewY you end up also modifying the _a_ value  of the matrix which is not desirable (took me some time to figure out this issue XD). Unfortunately, there are no native skew that performs a skewX and skewY simultaneously, so that is why I ended up defining it this way.